lib/cb/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cb
-  VERSION = '16.2.2'
+  VERSION = '17.0.0'
```

Comment:
I made it a breaking change in case anyone else was performing string manipulation as well. I was trying to be safe.